docs/source/use_cases/analyze_reads/calculate_coverage.rst

```diff
@@ -28,6 +28,9 @@ Setup Dataflow
 
 .. include:: ../../includes/dataflow_on_gce_setup.rst
 
+Your project must be whitelisted to use gRPC in order to run this pipeline, as it makes use of 
+gRPC streaming to retrieve data.
```

Comment:
Can probably use the same link for existing trusted tester endpoints:

google-genomics-contact@googlegroups.com

On Thu, Jun 11, 2015 at 12:57 PM, Nicole Deflaux <notifications@github.com>
wrote:

> In docs/source/use_cases/analyze_reads/calculate_coverage.rst
> <https://github.com/googlegenomics/start-here/pull/51#discussion_r32261039>
> :
>
> > @@ -28,6 +28,9 @@ Setup Dataflow
> >
> >  .. include:: ../../includes/dataflow_on_gce_setup.rst
> >
> > +Your project must be whitelisted to use gRPC in order to run this pipeline, as it makes use of
> > +gRPC streaming to retrieve data.
>
> Is there a link we can add here so that if people are interested, they can
> request to be whitelisted?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/googlegenomics/start-here/pull/51/files#r32261039>.
>